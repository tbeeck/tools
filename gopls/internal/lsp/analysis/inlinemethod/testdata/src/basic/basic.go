package main

import "fmt"

func PrintHello() {
	fmt.Println("Hello, 世界")
}

func main() {
	PrintHello() // want "inline method"
}
