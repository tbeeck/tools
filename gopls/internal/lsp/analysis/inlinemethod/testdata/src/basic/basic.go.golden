package main

import "fmt"

func PrintHello() {
	fmt.Println("Hello, 世界")
}

func main() {
	fmt.Println("Hello, 世界") // want "inline method"
}
